testsuite/MDAnalysisTests/test_tprparser.py

```diff
@@ -246,8 +246,8 @@ def _test_is_in_topology(name, elements, topology_section, topology_path):
     """
     universe = MDAnalysis.Universe(topology_path)
     for element in elements:
-        assert element in universe._topology[topology_section], \
-            'Interaction type "{0}" not found'.format(name)
+        if element not in universe._topology[topology_section]:
+            raise Exception('Interaction type "{0}" not found'.format(name))
```

Comment:
We do not want to raise `Exception`; this would cause the test to have an Error status rather a Fail one is the test fails. For a test to fail properly, the exception must be an `AssertError`.